test/Common/DurableTaskEndToEndTests.cs

```diff
@@ -3277,6 +3277,80 @@ public async Task TaskHubName_Throws_ArgumentException(string taskHubName)
                 argumentException.Message);
         }
 
+        /// <summary>

```

Comment:
In general, I think test names along with the test code and the comments within the code should be clear enough that you don't need to add lengthy comments at the method level for tests. This is because when running CI pipelines, you only get the Test name, not the comments associated with them.

Reply:
I agree. Tests should be easy to understand without comments (test are a form of documentation). What is the reason to specifically exclude a short test summary?

Reply:
There is nothing necessarily wrong with them, I just generally don't find them helpful most of the time, considering many of the times you get a failing test, you are just given the test name, not the test code (and comments). As long as the comment isn't used as a crunch and the test name is still a good one, then it shouldn't hurt.

Reply:
Just like comments within code, it would be meant to help (new) developers understand the test. Similarly to the test summary, you also don't see the test code or any within-code comments along with a failing test. I personally try to avoid comments unless absolutely necessary, but haven't thought of any reason to avoid summaries.

I do find it important to make sure tests are easy to understand without comments (and summaries). I'll take this opportunity to review my test to make sure it's clear, but leave the summary unless you object.